content/renderer/media/audio/mojo_audio_output_ipc.cc

```diff
@@ -233,6 +237,17 @@ void MojoAudioOutputIPC::Created(
       data_pipe->shared_memory;
   DCHECK(shared_memory_region.IsValid());
 
+#if defined(CASTANETS)
+  // If socket_handle is invalid.
+  if (socket_handle < 0) {
+    shared_memory_region_ = std::move(shared_memory_region);
```

Comment:
How about passing |shared_memory_region| with parameter to callback function without |shared_memory_region_| member?

Reply:
Thank you for your point.